src/custom_gates.jl

```diff
@@ -18,6 +42,130 @@ function matrix_rep(g::DoubleExcitation)
     return SMatrix{16,16,ComplexF64}(mat)
 end
 
+"""
+    DoubleExcitationPlus(ϕ)
+
+Generate the matrix representation of the [DoubleExcitationPlus](https://docs.pennylane.ai/en/stable/code/api/pennylane.DoubleExcitationPlus.html) gate.
+
+This gate performs an SO(2) rotation in the subspace {|1100⟩, |0011⟩} with a phase-shift on other states:
+
+- |0011⟩ ⟼ cos(φ/2)|0011⟩ - sin(φ/2)|1100⟩
```

Comment:
In the top line you have `ϕ` but in the equation, `\varphi` (this refuses to render for me)